example/ios/Flutter/Flutter.podspec

```diff
@@ -6,12 +6,12 @@
 Pod::Spec.new do |s|
   s.name             = 'Flutter'
   s.version          = '1.0.0'
-  s.summary          = 'High-performance, high-fidelity mobile apps.'
-  s.homepage         = 'https://flutter.io'
-  s.license          = { :type => 'MIT' }
+  s.summary          = 'A UI toolkit for beautiful and fast apps.'
+  s.homepage         = 'https://flutter.dev'
+  s.license          = { :type => 'BSD' }
```

Comment:
Maybe we could remove this file in git, since 
> This is a generated file; do not edit or check into version control.

Reply:
Yes. I will remove it.